mr/src/main/java/org/apache/iceberg/mr/mapreduce/IcebergInputFormat.java

```diff
@@ -396,8 +395,7 @@ public void close() throws IOException {
           throw new UnsupportedOperationException(
               String.format("Cannot read %s file: %s", file.format().name(), file.path()));
       }
-      currentCloseable = iterable;
-      return iterable.iterator();
+      return iterable;
```

Comment:
Same here. Could you add a newline after the control flow block?

Reply:
We follow the general style of Hadoop projects that I think is based on a Google style doc somewhere. We also add the rules to checkstyle when it can enforce them.